examples/openapi-javascript-wiki/.meshrc.yaml

```diff
@@ -3,6 +3,7 @@ sources:
     handler:
       openapi:
         oasFilePath: https://api.apis.guru/v2/specs/wikimedia.org/1.0.0/swagger.yaml
+        baseUrl: https://wikimedia.org/api/rest_v1
```

Comment:
@ardatan This wasn't defined before upgrading the handler, so how did the old handler got it?

Reply:
Actually I notices we didn't take `baseUrl` from swagger in the new one correctly. Now I fixed it so this addition is no longer needed.